blog/_posts/2016-08-20-hello-from-team-Crackers.md

```diff
@@ -0,0 +1,53 @@
+---
+title: hello from Team Crackers!!
+layout: post
+created_at: Sat August 20 2016
+permalink: blog/2016-08-20-hello-from-team-Crackers
+current: blog
+author: Nikita and Nishtha
+twitter: Team Crackers
+categories:
+- student-posts
+---
+
+# **Team Crackers**
+
+![Team Crackers](/img/blog/2016/teamCrackers1.jpg)    
```

Comment:
this image is not being found, because the filename is not correct. Could you change the file extension from `.JPG` to `.jpg` so that it works?